index.rst

```diff
@@ -269,6 +269,64 @@ our toolkit in the future (see how you can :ref:`get involved <contribute>`).
    </div>
 
 
+
+.. _started:
+
+Getting Started
+===============
+
+
+In order to start using the Fatiando tools you need to install a Python distribution. We recommend the `Anaconda Python distribution <https://www.anaconda.com/products/individual>`__. You can find detailed instructions on how to install it on any operating system in the `Software Carpentry setup instructions <https://carpentries.github.io/workshop-template/#python>`__.
+
+Once you finished setting up Anaconda, you can install all the latest Fatiando tools with a single command:
+
+.. raw:: html
+
+   <code>
+      conda install verde harmonica rockhound boule --channel conda-forge
+   </code>
+
+Alternatively, you can use the `pip package manager <https://pypi.org/project/pip/>`__:
+
+.. raw:: html
+
+   <code>
+      pip install verde harmonica rockhound boule
+   </code>
+
+`Fatiando's YouTube channel <https://www.youtube.com/channel/UCycNE5V984aAKL94GoA903g>`__
+has a tutorial on how to use Verde for gridding scattered data that may help you to get
+started with Fatiando tools:
+
+.. raw:: html
+
+   <div class="embed-responsive embed-responsive-16by9 youtube">
+      <iframe width="560" height="315" src="https://www.youtube.com/embed/-xZdNdvzm3E" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+   </div>
+
+
+.. raw:: html
+
+    <h2><i class="fas fa-rocket"></i> New to Python?</h2>
+
+If you don't have too much experience with Python, **don't worry!**.
+We recommend the `Software Carpentry lessons <https://software-carpentry.org/lessons/>`__, a great place to start learning Python for scientific computation.
+
+.. raw:: html
+
+   <ul class="fa-ul icon-list-small">
+      <li>
+         <i class="fa-li fa fa-book fa-fw"></i>
+         <a href="https://swcarpentry.github.io/python-novice-inflammation/">Programming with Python</a>
+      </li>
+      <li>
+         <i class="fa-li fa fa-book fa-fw"></i>
+         <a href="https://swcarpentry.github.io/python-novice-gapminder/">Plotting and Programming in Python</a>
+      </li>
+   </ul>
```

Comment:
Do we need to point to specific lessons here? Maybe just leave the link to the lessons page. What do you think?